Event Delegation/app.js

```diff
@@ -0,0 +1,7 @@
+let section = document.getElementsByTagName('section')[0];
```

Comment:
We can replace this with a `querySelector` so we don't need to access the `section` via index.

Reply:
Thanks, @lycoangelo, This is now updated.